CalibTracker/SiStripESProducers/plugins/services/modules.cc

```diff
@@ -40,3 +40,7 @@ DEFINE_FWK_SERVICE(SiStripBaseDelayGenerator);
 #include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
 #include "CalibTracker/SiStripESProducers/interface/SiStripConfObjectGenerator.h"
 DEFINE_FWK_SERVICE(SiStripConfObjectGenerator);
+
+#include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
+#include "CalibTracker/SiStripESProducers/interface/SiStripBadModuleFedErrService.h"
+DEFINE_FWK_SERVICE(SiStripBadModuleFedErrService);
```

Comment:
@suchandradutta can you please address David's comment to PR #13664  (https://github.com/cms-sw/cmssw/pull/13664#discussion_r58290500)